### ISSUE TITLE
isle2: a typo

### ISSUE THREAD

**Issue by deseven** [loiste-interactive/infra-issues#1172]
"atlest" in `papers_001_skin4`
![](http://eri.deseven.info/scr/IncuneationWebsterianUnifiedness.png)

**Comment by iarspider** [comment]
I would argue that this is not a bug, but rather an intended typo.

**Comment by CorruptComputer** [comment]
I agree, it looks hastily written, so it should probably have a mistake or two.

**Comment by deseven** [comment]
That's not the case. I asked one of developers to confirm that this is an unintentional typo before creating this issue.
So, it's a typo. An unintentional one.
